### PR TITLE
update new keyboard zoom65_lite

### PR DESCRIPTION
**Pull request by spbgzh**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by spbgzh** [comment]
@noroadsleft Thanks for your revise!

File: keyboards/meletrix/zoom65_lite/zoom65_lite.c [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "zoom65_lite.h"
+
+#ifdef VIA_ENABLE
```
Comment (drashna) [inline keyboards/meletrix/zoom65_lite/zoom65_lite.c:19]: Could this be moved to the via keymap, instead?

Comment (spbgzh) [reply keyboards/meletrix/zoom65_lite/zoom65_lite.c]: fiexed!

**Review by drashna — APPROVED** [review]

**Comment by spbgzh** [comment]
@drashna   Can you please give this PR another review？

File: keyboards/meletrix/zoom65_lite/rules.mk [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```
Comment (fauxpark) [inline keyboards/meletrix/zoom65_lite/rules.mk:16]:
```suggestion
```

Comment (spbgzh) [reply keyboards/meletrix/zoom65_lite/rules.mk]: Deleted

**Comment by spbgzh** [comment]
@fauxpark Sir can you check it, please?

File: keyboards/meletrix/zoom65_lite/rules.mk [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (fauxpark) [inline keyboards/meletrix/zoom65_lite/rules.mk:16]:
```suggestion
NKRO_ENABLE = yes           # Enable N-Key Rollover
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by spbgzh** [comment]
@fauxpark Solved)

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**